draft-foudil-securitytxt.md

```diff
@@ -46,33 +46,35 @@ researchers to follow in order to report security vulnerabilities.
 
 Many security researchers encounter situations where they are unable
 to report security vulnerabilities to organizations because there is
-no course of action laid out and no way indicated to contact the owner of a particular
-resource.
+no way indicated to contact the owner of a particular
+resource and no information available about the vulnerability disclosure practices
+of such owner.
 
 As per section 4 of {{?RFC2142}}, there is an existing convention
 of using the \<SECURITY@domain\> email address for communications regarding
 security vulnerabilities. That convention provides only a single, email-based
 channel of communication for security vulnerabilities per domain, and does not provide
 a way for domain owners to publish information about their security disclosure
-policies.
+practices.
 
 There are also contact conventions prescribed for Internet Service Providers (ISPs)
 in section 2 of {{?RFC3013}}, for Computer Security Incident Response Teams (CSIRTs)
 in section 3.2 of {{?RFC2350}} and for site operators in section 5.2 of
 {{?RFC2196}}. As per {{?RFC7485}}, there is also contact information provided by
 Regional Internet Registries (RIRs) and domain registries for owners of IP
 addresses, autonomous system numbers (ASNs) and domain names. However, none of
-these address the issue of how security researchers can locate disclosure
-policies and contact information for organizations in order to report
+these address the issue of how security researchers can locate vulnerability disclosure
+practices and contact information for organizations in order to report
 security vulnerabilities.
 
-In this document, we define a richer, machine-parsable and extensible way
+In this document, we define a richer and extensible way
 for organizations to communicate information about their security disclosure
-policies, which is not limited to email and also allows for additional features
+practices and ways to contact them, which is not limited to email and also allows for additional features
```

Comment:
This sentence is too wordy.

Reply:
Rewording this - being that we already mentioned this is intended for researchers to be consumed, I took that out and only left the part about the organizations.